scripts/analysis.py

```diff
@@ -845,17 +856,37 @@ def u_util_calc(cur):
     u_supply_timeseries = np.array(nat_u_timeseries(cur))
 
     # timeseries of fuel into reactors
-    fuel_timeseries = np.array(fuel_into_reactors(cur))
+    fuel_usage = np.array(fuel_into_reactors(cur))
 
     # timeseries of Uranium utilization
-    u_util_timeseries = np.nan_to_num(fuel_timeseries / u_supply_timeseries)
+    u_util_timeseries = np.nan_to_num(fuel_usage / u_supply_timeseries)
     print('The Average Fuel Utilization Factor is: ')
     print(sum(u_util_timeseries) / len(u_util_timeseries))
 
     return u_util_timeseries
 
 
-def where_comm(cur, commodity, prototypes, is_cum=True):
+def plot_uranium_utilization(cur):
+    """Plots uranium utilization factor of fuel cycle
+
+    Parameters
+    ----------
+    cur: sqlite cursor
+        sqlite cursor
+
+    Returns
+    -------
+    none
```

Comment:
There is a real `None` object in Python. This function does not return one. Can I recommend returning the plt object? Or, possibly the png file?